plugin-server/src/schema.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
A part of me is hesitant wrt adding ajv. Overall I think we should do this kind of stuff more and more but I wonder if we will end up stuck mangling json schemas... On the other hand it makes it interchangeable between services in the future.... Maybe I'm overthinking it 🤷

Reply:
If we find we're managing schemas without benefit we can just stop validating, no?

If we keep to "data we accept at the API" and "data we push to ClickHouse" then it _should_ stay as a manageable amount of schema